### DISCUSSION TITLE
Use Databricks Unity Catalog-stored table instead of Hive Metastore

### DISCUSSION THREAD

**Discussion by a-mases** [zinggAI/zingg#987] [Q&A] [answered]
Hello, I know it is possible to use Zingg with a delta table in the legacy hive metastore. How do we use a table that is stored instead with Unity Catalog? I tried feeding the catalog.schema.table path into inputPipe, as well as the external source link beginning with "s3://" and received a "could not read data" error both times. Thank you!

**Comment by sonalgoyal** [comment, marked as answer]
Unity catalog is not supported in the community version of Zingg

**Reply by a-mases** [reply]
Got it. Thank you!
